caddyhttp/browse/setup_test.go

```diff
@@ -66,4 +66,16 @@ func TestSetup(t *testing.T) {
 			}
 		}
 	}
+
+	// test case #6 tests startup with missing root directory in combination with default browse settings
+	controller := caddy.NewTestController("http", "browse")
+	cfg := httpserver.GetConfig(controller)
+
+	//Manipulate the root to a nonexistent directory, must be done, cause the default test root exists
```

Comment:
Could probably even simplify the comment to just: "Make sure non-existent root path doesn't return error"